app-backend/common/src/test/scala/com/implicits/Generators.scala

```diff
@@ -1149,7 +1149,8 @@ object Generators extends ArbitraryInstances {
       Gen.const(Nil),
       Gen.option(nonEmptyStringGen),
       Gen.const(true),
-      Gen.option(uuidGen)
+      Gen.option(uuidGen),
+      Gen.option(arbitrary[Int].map(_.toDouble))
```

Comment:
int -> double because the generator for doubles generates NaNs as well, which we don't need to handle here